.github/workflows/expire.yaml

```diff
@@ -0,0 +1,24 @@
+on:
+  # Once every 5 minutes is the max we can run a scheduled job.
+  schedule:
+    - cron: '5 * * * *'
```

Comment:
Not sure if 5 minutes is too often. Do you know if this will increase (and by how much) the monthly cost for GitHub?

What's the impact of cleaning-up less often?